Project/Source Files/Screens/SearchTab/SearchViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  SearchTabViewController.swift
+//  NetguruCollegeApp
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+
+	required init?(coder aDecoder: NSCoder) { fatalError("init(coder:) has not been implemented") }
```

Comment:
methods should always be declareds with closure in new line. so:
func x() {
sth()
}